.github/workflows/ci.yml

```
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10, 12, 13]
+        node-version: [12, 13]
```

Node 13 is itself unsupported, so we should drop that too. I'd like to see us go to this:

```suggestion
        node-version: [12, 14, 16]
```